src/pages/index.js

```diff
@@ -70,6 +158,12 @@ function openAddForm() {
 
 addPopup.setEventListeners();
 
+variables.profileAddButtonSelector.addEventListener("click", () =>
```

Comment:
`profileAddButtonSelector`

Word `selector` means a **string** that is used for searching elements in the DOM

**Please, correct it everywhere**